### PR TITLE
[docs] Add troubleshooting guide

### PR DESCRIPTION
**Pull request by aravindhp**
- Add a troubleshooting guide that describes access a Windows node and collect various log files
- Update the README.md with a strong recommendation to use the same  private key used to create the cluster in the cloud-private-key secret

### CODE REVIEW

**Review by christianh814 — COMMENTED** [review]
/lgtm

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Troubleshooting guide
+
+## WMCO does not go to running
+Please check if you are using an OKD/OCP 4.6 cluster running on Azure or AWS, configured with
+[hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md).
```
Comment (mansikulkarni96) [inline docs/TROUBLESHOOTING.md:5]: the link gives a 404 from the rich diff.

File: docs/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Windows Machine does not become a worker node
+There could various reasons as to why a Windows Machine does not become a worker node. Please collect the WMCO logs
+by executing:
+```shell script
+# oc logs $(oc get pods -o jsonpath={.items[0].metadata.name}) -n openshift-windows-machine-config-operator
```
Comment (mansikulkarni96) [inline docs/TROUBLESHOOTING.md:11]: should we remove the # prefix in the commands, it just creates a lighter font in the rich diff.

File: docs/TROUBLESHOOTING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```shell script
+# oc adm node-logs -l kubernetes.io/os=windows --path=/kubelet/kubelet.log
+```
+
+## How to collect Docker logs
+The Windows Docker service does not log to stdout but instead logs to the Windows' event log. You can view the Docker
+event logs using the following steps:
+* SSH into the Windows node and enter PowerShell:
+  ```shell script
+     <username>@<node-name> C:\Users\username> powershell
```
Comment (mansikulkarni96) [inline docs/TROUBLESHOOTING.md:64]: This line can be moved to the left, looks like an extra space.

**Review by mansikulkarni96 — COMMENTED** [review]
Thanks for working on this @aravindhp LGTM, PTAL at my comments, rest LGTM.

**Review by Anandnatraj — COMMENTED** [review]
Pls provide a link to these docs for additional Docker on Windows debugging
https://docs.microsoft.com/en-us/virtualization/windowscontainers/troubleshooting
https://docs.docker.com/docker-for-windows/troubleshoot/

**Comment by mansikulkarni96** [comment]
LGTM

**Comment by gmarkley-VI** [comment]
LGTM

**Pull request merged**